<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_AjoutSuppModifFournisseursEditeurs</class>
 <widget class="QWidget" name="F_AjoutSuppModifFournisseursEditeurs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>620</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="Rb_Fournisseur">
          <property name="text">
           <string>Fournisseur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Rb_Editeur">
          <property name="text">
           <string>Editeur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Rb_Les2">
          <property name="text">
           <string>Les deux</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="Lb_Recherche">
          <property name="text">
           <string>Rechercher un éditeur/fournisseur :</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLineEdit" name="LE_Recherche"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="TbV_Recherche">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="Bt_Ajouter">
          <property name="text">
           <string>Créer un éditeur/fournisseur</string>
          </property>
          <property name="icon">
           <iconset resource="ImagesIcones.qrc">
            <normaloff>:/BBD_Ajouter.png</normaloff>:/BBD_Ajouter.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="Lb_Nom">
          <property name="text">
           <string>Nom :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_Nom"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="Lb_Rue">
          <property name="text">
           <string>Rue et N°:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_Rue"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="Lb_CodePostal">
          <property name="text">
           <string>Code Postal:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_CodePostal"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="Lb_Ville">
          <property name="text">
           <string>Ville:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_Ville"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="Lb_Pays">
          <property name="text">
           <string>Pays:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_Pays"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="Lb_Contact">
          <property name="text">
           <string>Contact:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_Contact"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="Lb_Telephone">
          <property name="text">
           <string>Tel fixe :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_Telephone"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="Lb_Fax">
          <property name="text">
           <string>Fax:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_Fax"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="Lb_Email">
          <property name="text">
           <string>Email:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_Email"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="Lb_SiteWeb">
          <property name="text">
           <string>Site Web :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_SiteWeb"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="Lb_Remarques">
            <property name="text">
             <string>Remarques:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="TxE_Remarques"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCommandLinkButton" name="Bt_Valider">
          <property name="text">
           <string>Valider</string>
          </property>
          <property name="icon">
           <iconset resource="ImagesIcones.qrc">
            <normaloff>:/Valider.png</normaloff>:/Valider.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="Bt_Annuler">
          <property name="text">
           <string>Annuler </string>
          </property>
          <property name="icon">
           <iconset resource="ImagesIcones.qrc">
            <normaloff>:/Annuler.png</normaloff>:/Annuler.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="Bt_Supprimer">
        <property name="text">
         <string>Supprimer ce fournisseur</string>
        </property>
        <property name="icon">
         <iconset resource="ImagesIcones.qrc">
          <normaloff>:/BDD_Supprimer.png</normaloff>:/BDD_Supprimer.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ImagesIcones.qrc"/>
 </resources>
 <connections/>
</ui>
